certora/specs/Marketplace.spec

```diff
@@ -191,6 +228,9 @@ invariant cancelledSlotAlwaysHasCancelledRequest(env e, Marketplace.SlotId slotI
     currentContract.slotState(e, slotId) == Marketplace.SlotState.Cancelled =>
         currentContract.requestState(e, currentContract.slots(e, slotId).requestId) == Marketplace.RequestState.Cancelled;
 
+invariant requestsCountIsGreaterOrEqualToZero()
+    requestsCount >= 0;
```

Comment:
@gravityblast did we see counter examples where we had a negative request count?
Wondering otherwise how useful this invariant is.

Reply:
@0x-r4bbit  yes we had a run with `requestsCount` starting with `-1` because the `init_state axiom` works only with invariants, so as discussed with the Certora team we need this invariant to make sure it's not a negative number in the rule